src/tests/fake/subtractions.ts

```diff
@@ -77,6 +79,13 @@ export function createFakeSubtractionMinimal(overrides?: CreateFakeSubtractionMi
     return assign(defaultSubtractionMinimal, overrides);
 }
 
+/**
+ * Create a fake subtraction shortlist
+ */
+export function createFakeReducedSubtraction(): SubtractionShortlist {
+    return pick(createFakeSubtractionMinimal(), ["id", "name", "ready"]);
+}
```

Comment:
```suggestion
export function createFakeShortlistSubtraction(): SubtractionShortlist {
    return pick(createFakeSubtractionMinimal(), ["id", "name", "ready"]);
}
```
? Not stuck on it, since we have a `ShortlistSubtraction` type now this tracks for me